File: Danyltsiv/Java&Android/prac1PhoneDirectory/src/prac1PhoneDirectory/Record.java
package prac1PhoneDirectory;

public class Record {

	public String _Surname;
	public String _Forename;
    public String _Number;
    public String _Address;

    public Record(String Forename, String Surname,String Number, String Address)
    {
        this._Forename = Forename;
        this._Surname = Surname;
        this._Number = Number;
        this._Address = Address;
    }

    public String ToString(){

        return String.format("%s %s %s %s",this._Forename,this._Surname, this._Number, this._Address);
    }
    
}

File: Danyltsiv/C# Part 2/prac8task1TelephoneDirectory/Record.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TelephoneDirectory
{
    public class Record
    {
        public string _Surname;
        public string _Forename;
        public string _Number;
        public string _Address;

        public Record(string Forename, string Surname,string Number, string Address)
        {
            this._Forename = Forename;
            this._Surname = Surname;
            this._Number = Number;
            this._Address = Address;
        }

        public override string ToString()
        {
            return string.Concat(this._Forename," ",this._Surname, " ", this._Number, " ", this._Address);
        }
    }
}
